### ISSUE TITLE
Alt Text: Add Char Count, Allow Overflow

### ISSUE THREAD

**Issue by birdbree** [bluesky-social/social-app#1739]
**Is your feature request related to a problem? Please describe.**

This feature request aims to improve the ergonomics of alt text use significantly in cases where, for an example, OCR is being used to copy and paste text. Currently, overly long text is simply truncated at the character count maximum when pasted.

**Describe the solution you'd like**

Just like the post body editor, the alt text editor should feature a character count and allow alt text that spills over that count, indicating negative characters available as necessary.  This would allow a user to e.g. paste OCR'd text wholesale and then, if it exceeds the limit, pare the text down as necessary to fit.

**Describe alternatives you've considered**

I'm not sure there are any alternatives worth considering -- this merely brings the alt text editor to feature parity with the post body editor.

**Additional context**

N/A

**Comment by surfdude29** [comment]
This has now been implemented, there is a character counter for alt text and if it exceeds 2000 characters a warning is shown that it will be truncated:

![Image](https://github.com/user-attachments/assets/fbb2fb97-d6fd-4083-8a12-19fa59498bd3)

**Comment by cooljeanius** [comment]
> This has now been implemented, there is a character counter for alt text and if it exceeds 2000 characters a warning is shown that it will be truncated:
> 
> ![Image](https://github.com/user-attachments/assets/fbb2fb97-d6fd-4083-8a12-19fa59498bd3)

so is this in 1.96.5?

**Comment by surfdude29** [comment]
Yes, it's been in the app [since 1.92.0](github.com/bluesky-social/social-app/commit/76d63f9967a9a59193652487d2a10f41eac22268) was released in October.
